spec/lib/publishing_api_finder_publisher_spec.rb

```diff
@@ -109,12 +106,8 @@ def make_finder(base_path, overrides = {})
       end
     end
 
-    context "when the finder isn't `pre_production`" do
```

Comment:
Ideally this (and the removals below) should also be in the "Remove usages of pre_production flag" commit, but I appreciate it may be hard to unpick as at this point we've changed the deployment logic 🤔

Reply:
Similarly here - if we leave these tests here they would be failing